app/template/layout.php

```diff
@@ -49,7 +49,7 @@
     <script src="/assets/js/theme.js"></script>
 </head>
 
-<body hx-ext="head-support,loading-states" hx-boost="true" hx-indicator="#hx-indicator">
+<body hx-ext="head-support,loading-states" hx-boost="false" hx-indicator="#hx-indicator">
```

Comment:
Drop this change please 🙃 I assume it's an oversight

Reply:
Sure was